VideoCalls/RoomsTableViewController.m

```diff
@@ -265,6 +268,8 @@ - (void)setNavigationLogoButton
     } else {
         self.navigationItem.titleView = [[UIImageView alloc] initWithImage:logoImage];
     }
+    self.navigationItem.titleView.accessibilityLabel = @"Nextcloud Talk logo";
```

Comment:
If it’s the Nextcloud Talk logo, it can just say "Nextcloud Talk". :)

Reply:
Changed in https://github.com/nextcloud/talk-ios/commit/d784065529fa04745967e98396e15fceede81e5a! :)